turbinia/workers/photorec.py

```diff
@@ -44,21 +43,21 @@ def run(self, evidence, result):
     output_evidence = PhotorecOutput()
     # Create a path that we can write the new file to.
     base_name = os.path.basename(evidence.local_path)
-    output_file_path = os.path.join(self.output_dir, base_name)
+    output_file_path = os.path.join(self.output_dir, 'photorec_finding')
     # Add the output path to the evidence so we can automatically save it
     # later.
     output_evidence.local_path = output_file_path
-    photorec_log = os.path.join(self.output_dir, 'photorec.log')
     try:
       # Generate the command we want to run.
-      cmd = 'photorec /log /d {0:s} /cmd {1:s}  options, paranoid,keep_corrupted_fille,search'.format(
+      cmd = 'photorec /d {0:s} /cmd {1:s}  options,paranoid,keep_corrupted_fille,search'.format(
```

Comment:
Even if there is nothing useful in the log file for the report, I think we should still keep the log file as it might be useful for debugging.   If it's too verbose with the debug flag, we could also only add the debug flag if config.DEBUG is set.  This is used to turn on debugging logs in some of the other tasks as well.